sdk/iot/Azure.Iot.Hub.Service/src/Authentication/StaticSasTokenProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    public class StaticSasTokenProvider : ISasTokenProvider
```

Comment:
"Static" is kind of an overloaded term. We don't want people thinking that this class itself is static. Maybe "FixedSasTokenProvider" or "NonRenewableSasTokenProvider"?

Reply:
I'll rename it to `FixedSasTokenProvider`